<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_MoveOptions</class>
 <widget class="QWidget" name="LC_MoveOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="1">
    <widget class="QRadioButton" name="rbKeepOriginals">
     <property name="text">
      <string>Keep Originals</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="cbCurrentLayer">
     <property name="text">
      <string>Use Current Layer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="rbDeleteOriginals">
     <property name="text">
      <string>Delete Originals</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="sbNumberOfCopies">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="cbCurrentAttr">
     <property name="text">
      <string>Use Current Attributes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="rbMultipleCopies">
     <property name="text">
      <string>Multiple copies</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
